migrations/extract-data-record-content.js

```diff
@@ -0,0 +1,102 @@
+var _ = require('underscore'),
```

Comment:
This is still in the `migrations` directory? What's going to stop this from running automatically? Or do we want it to run automatically now?